packages/benchmark/package.json

```diff
@@ -1,13 +1,19 @@
 {
-  "private": true,
   "name": "@agentica/benchmark",
   "version": "0.7.0",
   "main": "src/index.ts",
+  "publishConfig": {
```

Comment:
https://pnpm.io/package_json#publishconfig
`pnpm publish` expand this property automatically when publishing